src/js/Store.js

```diff
@@ -52,9 +50,7 @@ const store = {
     },
 
     componentsListeningForEvent(event) {
-        return this.components().filter(component => {
-            return component.events.includes(event)
-        })
+        return this.components().filter(({ events }) => events.includes(event))
```

Comment:
yeah, I think for me, I like seeing "component" as the param (even if it's obvious because it's chained off a filter), it just reads better to me. But like I said, I like the single return.